server/enterprise/metrics/metrics.go

```diff
@@ -1162,6 +1162,7 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 			Subsystem: MetricsSubsystemClientsWeb,
 			Name:      "largest_contentful_paint",
 			Help:      "Duration of how long it takes for large content to be displayed on screen to a user (seconds)",
+			Buckets:   append(prometheus.DefBuckets, 15, 20),
```

Comment:
[`append`](https://pkg.go.dev/builtin#append) mutates `prometheus.DefBuckets`. Should we use [`prometheus.ExponentialBucketsRange`](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#ExponentialBucketsRange) to define a new slice instead? Or use the new `slices.Clone` and append to that?

Reply:
Very good catch. I expected `append` to always make a copy of the slice. I'll definitely fix this